src/Common/EnvironmentProxyConfigurationResolver.cpp

```diff
@@ -0,0 +1,30 @@
+#include "EnvironmentProxyConfigurationResolver.h"
+
+#include <Common/logger_useful.h>
+#include <Poco/URI.h>
+
+namespace DB
+{
+
+static constexpr auto PROXY_HTTP_ENVIRONMENT_VARIABLE = "http_proxy";
```

Comment:
Usually environment variables are upper-case, is there any reason to have those lower-case?

Reply:
Not a strong reason, I just happened to come across more lowercase examples than uppercase ones for proxy env variables. Some [sources](https://unix.stackexchange.com/a/212972) also suggest `proxy` environment variables are an exception and are usually lowercase